cmd/go-clang-gen/main_array_test.go

```diff
@@ -0,0 +1,55 @@
+package main
```

Comment:
Took me a while to figure out why this is in the cmd folder and not in /api_test.go. Please rename the file to "main_test.go" since we want to test more than just arrays with this.